app/src/main/res/layout/dialog_sleep_timer.xml

```diff
@@ -1,28 +1,41 @@
 <?xml version="1.0" encoding="utf-8"?>
-<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:seekarc="http://schemas.android.com/apk/res-auto"
     android:layout_width="match_parent"
-    android:layout_height="wrap_content">
+    android:layout_height="wrap_content"
+    android:orientation="vertical">
 
-    <com.triggertrap.seekarc.SeekArc
-        android:id="@+id/seek_arc"
-        android:layout_width="match_parent"
-        android:layout_height="wrap_content"
-        android:layout_gravity="center"
-        android:padding="30dp"
-        seekarc:SeekArc_clockwise="false"
-        seekarc:SeekArc_max="120"
-        seekarc:SeekArc_progressColor="?colorAccent"
-        seekarc:SeekArc_rotation="180"
-        seekarc:SeekArc_startAngle="30"
-        seekarc:SeekArc_sweepAngle="300" />
+    <FrameLayout
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content">
+
+        <com.triggertrap.seekarc.SeekArc
+            android:id="@+id/seek_arc"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:layout_gravity="center"
+            android:padding="30dp"
+            seekarc:SeekArc_clockwise="false"
+            seekarc:SeekArc_max="120"
+            seekarc:SeekArc_progressColor="?colorAccent"
+            seekarc:SeekArc_rotation="180"
+            seekarc:SeekArc_startAngle="30"
+            seekarc:SeekArc_sweepAngle="300" />
+
+        <TextView
+            android:id="@+id/timer_display"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_gravity="center"
+            android:textAppearance="?android:textAppearanceLarge" />
+    </FrameLayout>
 
-    <TextView
-        android:id="@+id/timer_display"
+    <CheckBox
+        android:id="@+id/checkBox"
         android:layout_width="wrap_content"
-        android:layout_height="wrap_content"
-        android:layout_gravity="center"
-        android:textAppearance="?android:textAppearanceLarge" />
+        android:layout_height="42dp"
+        android:layout_gravity="bottom|center_horizontal"
```

Comment:
I think it's fine if it's aligned to the start (i.e. left for English) instead of centred.